src/components/m-policy-feature-item/_template.js

```diff
@@ -0,0 +1,10 @@
+import html from 'nanohtml';
+
+export default ({ src, title, description }) => html`
+  <div class="m-policy-feature-item__content">
+    <img class="m-policy-feature-item__icon" src="${src}" alt="Policy feature icon">
```

Comment:
Why did you not use the axa-icon? You can add new design-approved icons in the spritesheet either here or if is only relevant for AEM, just there

We have an icon spritesheet in the patterns library and the one in AEM is also compatible with the one here. So question is: If we need SVG and they are icons (as they are now in aem), please add the spritesheet (call me maxbe so i can show you).

If instead they are meant to be images that content author uplod, then `img` is ok

Reply:
these are images (in our case SVGs that the AEM author uploads into dam), so I think it's ok

Reply:
ok good for me then :)

Reply:
@nakovskijosif please add a variable for the alt text: alt="Policy feature icon"

Reply:
Alt variable added. Thank you for your feedback.